SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Supported versions which accept security updates can be found using the next link: https://docs.janusgraph.org/changelog/#currently-supported
+
+## Reporting a Vulnerability
+
+To report private security vulnerabilities, please, send email to the following address: **security (at) janusgraph.org**  
+
+To report public security vulnerabilities, please, open a PR with `kind/security` label.
```

Comment:
Should we even have this option here? Perhaps we should steer folks to reach out to us privately instead? 

Otherwise, if there are security issues submitted but not fixed immediately, that's just publicizing security vulnerabilities in JanusGraph that could affect existing production deployments.